contents/teams/growth/objectives.mdx

```diff
@@ -1,7 +1,15 @@
-- **Objective 1**: Double activation rates for product analytics and session replay
-  - Reasoning: Activation is lower than we'd like it to be, and activation is a leading indicator for retention and revenue
-  - **KR**: 2 customer interviews per week
-  - **KR**: Start 1 new experiment every week
-- **Objective 2**: Launch pricing for Data warehouse, queries, CDP
-  - Reasoning: Customers get lots of value out of these and should be charged accordingly
-  - **KR**: Cost of collaboration doesn't feel too high
\ No newline at end of file
+### Q2 2024 Objectives
+
+- **Objective 1**: Create a flow in product analytics onboarding to fill out a dashboard template using actions (Raquel)
```

Comment:
Should we extend this to define all the activation definitions?

Reply:
Historically we've kept some of our business metrics private so I intentionally left them out here.